compiler/rustc_codegen_ssa/src/mir/statement.rs

```diff
@@ -115,6 +115,30 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
                 self.codegen_coverage(&mut bx, coverage.clone());
                 bx
             }
+            mir::StatementKind::CopyNonOverlapping(box mir::CopyNonOverlapping {
+                ref src,
+                ref dst,
+                ref size,
+            }) => {
+                let dst_val = self.codegen_operand(&mut bx, dst);
+                let src_val = self.codegen_operand(&mut bx, src);
+                let size_val = self.codegen_operand(&mut bx, size);
+                let size = size_val.immediate_or_packed_pair(&mut bx);
+                let dst = dst_val.immediate_or_packed_pair(&mut bx);
+                let src = src_val.immediate_or_packed_pair(&mut bx);
+                use crate::MemFlags;
+                let flags =
+                    (!MemFlags::UNALIGNED) & (!MemFlags::VOLATILE) & (!MemFlags::NONTEMPORAL);
```

Comment:
This is setting every memflag except for the specified ones. There may be memflags currently or on the future that shouldn't be used. Maybe 0 is enough?

Reply:
I changed it to `MemFlags::empty()`, which presumably is 0 but can be adjusted in the future.